packages/loot-core/src/types/server-handlers.d.ts

```diff
@@ -121,7 +121,7 @@ export interface ServerHandlers {
   'payees-get-rules': (arg: { id: string }) => Promise<RuleEntity[]>;
 
   'make-filters-from-conditions': (arg: {
-    conditions;
+    conditions: unknown;
```

Comment:
I think this could be a RuleConditionEntity and the return type is the type of the parameter of Query.filter function

Reply:
That's a very good suggestion, but unfortunately it will create a lot of failures downstream (that I would then need to patch). So IMO we can pick that up later - in a separate PR - instead of trying to cram everything in this one :)